vpython/vpython.py

```diff
@@ -3138,17 +3138,15 @@ def pixel_to_world(self):
     def pixel_to_world(self, value):
         raise AttributeError('pixel_to_world is read-only')
 
-    def capture(self, *s):
-        if len(s) == 0:
-            raise AttributeError('scene.capture requires at least one argument.')
-        filename = s[0]
-        if not isinstance(filename, str): raise AttributeError('A capture file name must be a string.')
-        if '.png' not in filename: filename += '.png'
-        include_labels = "T"
-        if len(s) == 2:
-            if s[1] == True: include_labels = "T"
-            else: include_labels = "F"
-        self.addmethod('capture', include_labels+filename)
+    def capture(self, filename, capture_labels=True):
+        if not isinstance(filename, str):
+            raise TypeError("'filename' for Capture must be a string.")
+
+        if filename[-4:] != ".png":
```

Comment:
We may would want to warn user about it

Reply:
tho they will understand this behaviour if they will just play around with this function

Reply:
What do you think about changing  `filename[-4:] != ".png"` to `filename.endswith(".png")`? The second one is a little easier for me to read but they have the same effect. If you prefer to keep it as written, that is fine.

Reply:
> What do you think about changing `filename[-4:] != ".png"` to `filename.endswith(".png")`?

i thought that `str.endswith` is relatively new functionality, but actually it was introduced in python 3.3, so i think we are free to use it here

Reply:
I'll keep an eye out for another commit with that change 😃

Reply:
[resolved](https://github.com/vpython/vpython-jupyter/pull/165/commits/2eb35114db37be7685a5f7286da05d75b237ea90)